setup.py

```diff
@@ -30,6 +30,7 @@
 tests_require = [
     'mock; python_version < "3.0"',
     'nose',
+    'aiounittest',
 ]
```

Comment:
There goes all my hard work keeping imports, `__all__`, etc. in alphabetical order... 😹

Reply:
hahaha sorry, didn't notice :smile_cat: 
gonna fix this!